### PR TITLE
Fix packaging to include all project files

### LINKED ISSUE #60: Crash if not installed in editable mode
**Description:**

When installed with `pip install path/to/stubdefaulter/repo`, `stubdefaulter` crashes on this line. 
https://github.com/JelleZijlstra/stubdefaulter/blob/afee42c3a783f0bd92179691653d4ca12f1d56f3/stubdefaulter.py#L550
This is because the file `stdlib-blacklist.txt` is not packaged in the distribution that gets installed with the pip command above.

**Workaround:**

Install in *editable mode* `pip install -e path/to/stubdefaulter/repo`. In this case `stubdefaulter` is invoked in-tree from the git clone where the `stdlib-blacklist.txt` file exists.

**Solution:**

Any of:
1. package the `stdlib-blacklist.txt` file with the project;
2. change the usage entry in the README from `pip install .` to `pip install -e .`
3. do nothing, this is not a supported.

### PR DESCRIPTION
**Pull request by hamdanal**
Closes #60

Now that multiple files have to be included, I made it into a package so that all files are installed under the namespace of the project.

### CODE REVIEW

**Comment by hamdanal** [comment]
What's packaged:

 sdist:
```console
$ tar -tf dist/stubdefaulter-0.0.0.tar.gz
stubdefaulter-0.0.0/stubdefaulter/__init__.py
stubdefaulter-0.0.0/stubdefaulter/__main__.py
stubdefaulter-0.0.0/stubdefaulter/stdlib-blacklist.txt
stubdefaulter-0.0.0/.gitignore
stubdefaulter-0.0.0/pyproject.toml
stubdefaulter-0.0.0/PKG-INFO
```

wheel
```console
$ unzip -l dist/stubdefaulter-0.0.0-py3-none-any
.whl
Archive:  dist/stubdefaulter-0.0.0-py3-none-any.whl
  Length      Date    Time    Name
---------  ---------- -----   ----
    22244  2020-02-02 00:00   stubdefaulter/__init__.py
       70  2020-02-02 00:00   stubdefaulter/__main__.py
     2070  2020-02-02 00:00   stubdefaulter/stdlib-blacklist.txt
      718  2020-02-02 00:00   stubdefaulter-0.0.0.dist-info/METADATA
       87  2020-02-02 00:00   stubdefaulter-0.0.0.dist-info/WHEEL
       53  2020-02-02 00:00   stubdefaulter-0.0.0.dist-info/entry_points.txt
      578  2020-02-02 00:00   stubdefaulter-0.0.0.dist-info/RECORD
---------                     -------
    25820                     7 files
```

File: pyproject.toml [hunk 1]
```diff
@@ -24,3 +24,6 @@ dev = [
     "pre-commit-hooks==4.4.0",   # Must match .pre-commit-config.yaml
     "pytest",
 ]
+
+[tool.hatch.build]
+packages = ["stubdefaulter"]
```
Comment (hamdanal) [inline pyproject.toml:29]: Otherwise hatchling includes all types of cruft in the sdist like the venv folder and pytest cache etc.

**Review by JelleZijlstra — APPROVED** [review]

**Comment by JelleZijlstra** [comment]
Seems like you need to change the pyanalyze test to run on `stubdefaulter/` instead of `stubdefaulter.py`.

**Comment by hamdanal** [comment]
BTW would you consider adding a license? I just realized the project doesn't have any. Technically it cannot be used by others.

**Comment by JelleZijlstra** [comment]
Good point, I'll add a license. Would it also be useful to you if I publish on PyPI?

**Comment by hamdanal** [comment]
> Good point, I'll add a license. Would it also be useful to you if I publish on PyPI?

It would save me some key strokes when I pip install but do not bother if you don't want to, not a big deal. I only tried it once with some [stubs](https://github.com/hamdanal/python-stubs) I keep around for a better vscode experience (unsuccessfully for now but I haven't spent more than 5 minutes on it yet).

Thank you for creating this btw, it helps a lot with editors that rely on typeshed for showing function and class signatures.

**Pull request merged**